### ISSUE TITLE
Multus Readme, change cniVersion for Kubernetes 1.16

### ISSUE THREAD

**Issue by lubouski** [k8snetworkplumbingwg/multus-cni#415]
**Readme Documentation change**:
`At least Here` https://github.com/intel/multus-cni/blob/master/doc/how-to-use.md#networkattachmentdefinition-with-json-cni-config
cniVersion should be changed, errors observed in logs
```
[plugin multus does not support config version ""]
```

**In new Kubernetes release they changed API a bit**:
So we need to make assumption in docs that for `Kubernetes 1.16` - `cniVersion` should be changed to `0.3.1` from `0.3.0`

**Comment by electrocucaracha** [comment]
Last week [Kubespray](https://kubespray.io/) has released its latest stable release (v2.12.0) which deploys Kubernetes v1.16.3.  The `kube_network_plugin_multus` configuration value enables Multus CNI installation. This tool uses the [nfvpe Docker images](https://hub.docker.com/r/nfvpe/multus) for its deployment and the `multus_image_tag` configuration values to specify the version to be used which for this release is v3.2.1. 

If Multus is enabled, the `00-multus.conf` file created by the [`kube-multus-ds-amd64`](https://github.com/kubernetes-sigs/kubespray/blob/v2.12.0/roles/network_plugin/multus/templates/multus-daemonset.yml.j2) Daemonset will have the following content for flannel.

```
{
  "name": "multus-cni-network",
  "type": "multus",
  "kubeconfig": "/etc/cni/net.d/multus.d/multus.kubeconfig",
  "delegates": [
    {
      "name": "cni0",
      "cniVersion": "0.3.1",
      "plugins": [
        {
          "type": "flannel",
          "delegate": {
            "forceAddress": true,
            "hairpinMode": true,
            "isDefaultGateway": true
          }
        },
        {
          "type": "portmap",
          "capabilities": {
            "portMappings": true
          }
        }
      ]
    }
  ]
}
```
This configuration is only causing errors in Kubelet when Kubernetes v1.16.3 is deployed.
```
[plugin multus does not support config version ""]
```
I'm wondering which is the right configuration value or version to be used for Kubernetes v1.16.3

**Comment by holmesb** [comment]
CNI version v0.3.1 and Multus v3.4 [not working](https://github.com/kubernetes-sigs/kubespray/pull/5493#issuecomment-596179041) on k8s v1.16.7
